### PR TITLE
Add pytest process for outputting coverage or other artifacts

### PR DESCRIPTION
**Pull request by stuart23**
There are no clear instructions on how to do this. See https://github.com/astronomer/astro-cli/issues/1142

### CODE REVIEW

**Comment by frankcash** [comment]
https://docs.pytest.org/en/7.1.x/how-to/output.html#creating-junitxml-format-files

**Review by frankcash — APPROVED** [review]

File: astro/cli/astro-dev-pytest.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ hide_table_of_contents: true
 
 Run unit tests for your data pipelines on Astro with `pytest`, a testing framework for Python. When you run this command, the Astro CLI creates a local Python environment that includes your DAG code, dependencies, and Astro Runtime Docker image. The CLI then runs any pytests in the `tests` directory of your Astro project and shows you the results of those tests in your terminal.
 
+This command runs `pytest` in a container. If your test is generating artifacts that you need to use (e.g. code coverage reports), these can be output to the `include` directory in the container so they can be accessed after the test has finished.
```
Comment (jwitz) [inline astro/cli/astro-dev-pytest.md:11]:
```suggestion
The command runs `pytest` in a container. If your test generates artifacts, such as code coverage reports, you can output the artifacts to the `include` folder of your Astro project so they can be accessed after the test has finished.
```
Is this right? If not, how would you access the `include` directory if it was in a container?

**Review by jwitz — CHANGES_REQUESTED** [review]
I made a suggestion that might include a change in technical meaning, so requesting changes to make sure that's all good before merging!

File: astro/cli/astro-dev-pytest.md (hunk 1 above)
Comment (stuart23) [reply astro/cli/astro-dev-pytest.md:11]: Exactly! The include dir is shared inside and outside the container and is therefore the only place we can pass stuff out of the test environment. Your changes look good

**Review by jwitz — APPROVED** [review]

**Pull request merged**